jcl/src/java.base/share/classes/openj9/internal/tools/attach/target/DiagnosticUtils.java

```diff
@@ -485,5 +523,13 @@ private static DiagnosticProperties doHelp(String diagnosticCommand) {
 
 		commandTable.put(DIAGNOSTICS_LOAD_JVMTI_AGENT, DiagnosticUtils::loadJVMTIAgent);
 		helpTable.put(DIAGNOSTICS_LOAD_JVMTI_AGENT, DIAGNOSTICS_LOAD_JVMTI_AGENT_HELP);
+
+/*[IF CRAC_SUPPORT]*/
```

Comment:
instead you can add a check here to see if Crac is enabled before adding the commands

Reply:
Added the check `InternalCRIUSupport.isCRaCSupportEnabled()`.